ui/shared/status/buttons/StatusIconTabButton.qml

```diff
@@ -0,0 +1,81 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+import "../../../imports"
+import "../../"
+import "../core"
+import "../"
+
+TabButton {
+    id: control
+
+    property color iconColor: Style.current.secondaryText
+    property color disabledColor: iconColor
```

Comment:
@borismelnik we should start talking about theming and colors. Certainly not a good idea to use `secondaryText` here.